src/lib.rs

```diff
@@ -192,6 +192,23 @@ pub enum MouseCursor {
     RowResize,
 }
 
+#[derive(Clone, PartialEq)]
+pub enum FullScreenState {
+    None,
+    Windowed,
+    Total(platform::MonitorId),
```

Comment:
Probably, I don't have much experience with this kind of API to know what's usually used.